src/patternfly/components/Toggle/toggle.scss

```diff
@@ -0,0 +1,181 @@
+.pf-c-toggle {
+  --pf-c-toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-toggle--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-toggle--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-toggle--Color: var(--pf-global--Color--100);
+  --pf-c-toggle--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-toggle--BackgroundColor: transparent;
+
+  // border
+  --pf-c-toggle--after--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--after--BorderRightWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--after--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--after--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--after--BorderTopColor: var(--pf-global--BorderColor--300);
+  --pf-c-toggle--after--BorderRightColor: var(--pf-global--BorderColor--300);
+  --pf-c-toggle--after--BorderBottomColor: var(--pf-global--BorderColor--200);
+  --pf-c-toggle--after--BorderLeftColor: var(--pf-global--BorderColor--300);
```

Comment:
Can you split the gray border and the accent border with `::before` (being the gray border) and `::after` (being the accent border) like we've done in other places, table, tabs, etc, to avoid a bevel if user updates `border-width`?

![Screen Shot 2021-02-09 at 10 12 19 AM](https://user-images.githubusercontent.com/5385435/107383998-8ce44500-6abf-11eb-87a6-a2aafe07950c.png)

Reply:
Updated